src/api/endpoints.ts

```diff
@@ -15,6 +15,8 @@ export const endpoints = {
   DEACTIVATE_USER: `${process.env.REACT_APP_AUTHORIZATION_API_BASE}institution-users/deactivate`,
   ACTIVATE_USER: `${process.env.REACT_APP_AUTHORIZATION_API_BASE}institution-users/activate`,
   VENDORS: `${process.env.REACT_APP_TRANSLATION_ORDER_API_BASE}vendors`,
+  CREATE_VENDORS: `${process.env.REACT_APP_TRANSLATION_ORDER_API_BASE}vendors/bulk`,
```

Comment:
Since these two endpoints are the same, maybe we can use one key, rather than two?